quam_builder/architecture/superconducting/qubit/fixed_frequency_transmon.py

```diff
@@ -18,29 +17,3 @@ class FixedFrequencyTransmon(BaseTransmon):
     """
 
     xy_detuned: Union[MWChannel, IQChannel] = None  # TODO: should probably belong into qubit pairs since it is for ZZ
-
-    @property
-    def name(self):
-        """The name of the transmon"""
-        return self.id if isinstance(self.id, str) else f"q{self.id}"
-
-    def __matmul__(self, other):
-        if not isinstance(other, FixedFrequencyTransmon):
-            raise ValueError(
-                "Cannot create a qubit pair (q1 @ q2) with a non-qubit object, " f"where q1={self} and q2={other}"
-            )
-
-        if self is other:
-            raise ValueError("Cannot create a qubit pair with same qubit (q1 @ q1), where q1={self}")
-
-        for qubit_pair in self._root.qubit_pairs.values():
-            if qubit_pair.qubit_control is self and qubit_pair.qubit_target is other:
-                return qubit_pair
-        else:
-            raise ValueError("Qubit pair not found: qubit_control={self.name}, " "qubit_target={other.name}")
-
-    def align(self):
-        align(self.xy.name, self.resonator.name)
-
-    def wait(self, duration):
-        wait(duration, self.xy.name, self.resonator.name)
```

Comment:
The wait isn't a native method of the Qubit component, right?
Which means that now we don't have the ability to make all the elements of a qubit wait together

Reply:
It is a native method of the qubit component

Reply:
Are you sure? I saw the align but not the wait command

Reply:
actually, I realized it wasn't but shouldn't be, I reintroduced it, this time to BaseTransmon.wait

Reply:
Actually BaseTransmon doesn't work, because it won't include the flux line for flux tunable transmon, no?

Reply:
It collects all the channels

Reply:
Oh ok nice!